tensorflow/core/transforms/remapper/pass.cc

```diff
@@ -91,18 +96,227 @@ class MatchMulSigmoid : public RewritePattern {
   OperationName sigmoid_name_;
 };
 
+namespace {
+class RewriterBase : public RewritePattern {
```

Comment:
How about a name specific to remapper? Like RemapperPatternBase

Reply:
Renamed as suggested.